### ISSUE TITLE
Disco

### ISSUE THREAD

**Issue by akloeckner** [ZNEGGE-SDK/magichome#3]
Hey, I couldn’t think of any other way to contact you, so here we go. ;-)

I wondered, whether you might want to build on your partnership with lednet and improve/extend this home assistant integration together. I’m thinking  especially of the music-to-light “disco” feature. It would be really nice to be able to trigger it via home assistant...

@LIULi-VVET FYI

**Comment by LIULi-VVET** [comment]
Do you mean using the music mode function through homeassistant? @akloeckner

**Comment by akloeckner** [comment]
Yes, that's what I meant. I understand that actual music mode will be difficult, because home assistant might not have a microphone available. But maybe it can be emulated easily with a fixed beat rate. Or something similar.

**Comment by LIULi-VVET** [comment]
It’s a bit complicated, I need some time to see if it works
